### ISSUE TITLE
Disable directory browsing 

### ISSUE THREAD

**Issue by akhilleusuggo** [WWBN/AVideo-Storage#24]
https://github.com/WWBN/AVideo-Storage/blob/2e6dba59b16581a959f92312c8f43700d37745e3/.htaccess#L14

This line is giving me too many error logs, Removing the "menus" fixes my problem.

 AH01276: Cannot serve directory /var/www/html/AVideo-Storage/videos/xxxxxxxxxxx65.92760120/: No matching DirectoryIndex (index.html,index.cgi,index.pl,index.php,index.xhtml,index.htm) found, and server-generated directory index forbidden by Options directive

This issue is related to https://github.com/WWBN/AVideo/issues/5680

**Comment by DanielnetoDotCom** [comment]
Disable directory browser is very important for security reasons.

**Comment by akhilleusuggo** [comment]
then there's something wrong with the script. 

`AH01276: Cannot serve directory /var/www/html/AVideo-Storage/videos/xxxxxxxxxxx65.92760120/: No matching DirectoryIndex (index.html,index.cgi,index.pl,index.php,index.xhtml,index.htm) found, and server-generated directory index forbidden by Options directive`

**Comment by DanielnetoDotCom** [comment]
but there is no reason to access the videos directory without a file
why do you want to access that?

**Comment by akhilleusuggo** [comment]
@DanielnetoDotCom is the Local Server who tries to access to pull the tags data.
This error spams when you refresh almost any page having the tags option enabled.

**Comment by DanielnetoDotCom** [comment]
"Local Server who tries to access to pull the tags data." I am not sure what that means

**Comment by akhilleusuggo** [comment]
@DanielnetoDotCom The server checks the maximum resolutions of the videos

![image](https://user-images.githubusercontent.com/52116577/133685568-f7a0cd58-5d02-4985-a6c0-d7db08e4dc03.png)

**Comment by DanielnetoDotCom** [comment]
I am also using this configuration and I cannot see this error on my side.

**Comment by DanielnetoDotCom** [comment]
I just realize you are talking about the Storage, sorry about that.

I will review this correctly

**Comment by akhilleusuggo** [comment]
Yes, the Storage not local. I encoded them via normal encoding. No embed, just default ffmpeg encoding via the encoder, and stored on YPT-Storage.
